### DISCUSSION TITLE
How to trigger workflowtemplate in cron workflow

### DISCUSSION THREAD

**Discussion by pockyhe** [argoproj/argo-workflows#9569] [Q&A]
Hi, I'm trying to a workflow referred by a summited workflowtemplate in Cronworkflow. After have submitted this cronworkflow, It seems that the cronworkflow works well, but It cannot create a new workflow once scheduled time comes. My Cronworkflow is the following:
 
![image](https://user-images.githubusercontent.com/64390829/189475589-4fe03536-b9a0-4b1a-809d-5ec5e36b2583.png)

Do you have any ideas about how to solve it? I'm not sure if there is some error in my cronworkflow.

Thank you very much

**Comment by johnhamelink** [comment]
I was able to do this like so:

```
spec:
  workflowSpec:
    arguments: {}
    workflowTemplateRef:
      name: workflow-template-name-goes-here
```

Notice that I've not set an entrypoint. This example confirms the strategy is acceptable: https://github.com/argoproj/argo-workflows/blob/master/examples/workflow-template/workflow-template-ref.yaml

I figured this out by using the fields reference document to explore the cronworkflow's structure: https://argoproj.github.io/argo-workflows/fields/#cronworkflow. I found the example by looking through the "Examples with this field" examples in the documentation. 

I hope that helps!
